frontend/src/routes/dashboard/index.tsx

```diff
@@ -9,6 +9,7 @@ import { useExampleQuery } from "../../api/queries/useExampleQuery"
 
 /***** UTILITIES *****/
 import { useBrowserNotification } from "../../utilities/hooks/useBrowserNotification"
+import { Sidebar } from "../../components/InnerNavbar"
```

Comment:
Please move under a code comment titled `/***** SHARED *****/`

Reply:
Looks like this hasn't been actioned. This needs to be moved into the group under line 5 so that it is logically imported with the other "shared" components